cmd/gpu_plugin/README.md

```diff
@@ -224,9 +224,32 @@ master
 
 ## Testing and Demos
 
-We can test the plugin is working by deploying an OpenCL image and running `clinfo`.
-The sample [intel-opencl-icd](../../demo/intel-opencl-icd/) OpenCL image can be built
-using `make intel-opencl-icd` and must be made available in the cluster.
+We can test the plugin is working by deploying an [OpenCL image](../../demo/intel-opencl-icd/) that is running `clinfo`.
```

Comment:
Maybe:
> Working of the GPU plugin can be verified by deploying an  [OpenCL image](../../demo/intel-opencl-icd/) which runs `clinfo` outputting the GPU capabilities (detected by driver installed to the image).

Reply:
"suggestion tool"?  (no not see such option in review, and quick googling did not return relevant hits)

Reply:
Instead of "Working of " I chose "The GPU plugin functionality can be.."